cogs/SquadQueue.py

```diff
@@ -811,11 +968,14 @@ def allowed_players_check(players):
                 return mk8dx_players_allowed(players, self.room_mmr_threshold)
 
         all_confirmed_players = mogi.players_on_confirmed_teams()
-        first_late_player_index = (mogi.num_players//mogi.players_per_room) * mogi.players_per_room
+        first_late_player_index = (
+            mogi.num_players//mogi.players_per_room) * mogi.players_per_room
         regular_player_list = all_confirmed_players[:first_late_player_index]
         late_player_list = all_confirmed_players[first_late_player_index:]
-        proposed_list = sorted(mogi.generate_proposed_list(allowed_players_check), reverse=True)
-        await mogi.populate_host_fcs()
+        proposed_list = sorted(mogi.generate_proposed_list(
+            allowed_players_check), reverse=True)
+        if common.SERVER is common.Server.MKW:
+            await mogi.populate_host_fcs()
```

Comment:
This change is not strictly needed. See the contents of `Mogi.populate_host_fcs()`

Reply:
okay you right

Reply:
[https://github.com/emilp-595/QueueBot/pull/15/commits/4ff06800eb0913a7d89a56f6439019071b73aeff](https://github.com/emilp-595/QueueBot/pull/15/commits/4ff06800eb0913a7d89a56f6439019071b73aeff)